cmds/bmp/tasks_command.go

```diff
@@ -51,5 +55,41 @@ func (cmd tasksCommand) Validate() (bool, error) {
 
 func (cmd tasksCommand) Execute(args []string) (int, error) {
 	cmd.printer.Printf("Executing %s comamnd: args: %#v, options: %#v", cmd.Name(), args, cmd.options)
+
+	var latest uint
+	latest = 50
```

Comment:
I wonder if we can pass this as a optional parameter to the command and default to 50?